### PR TITLE
ROS logging

### PR DESCRIPTION
**Pull request by sah-huawei**
Added basic utils to route standard logging msgs to rospy.log.
Also quieted ros frequency warning in `ros_driver.py`.

### CODE REVIEW

File: examples/ros/src/smarts_ros/scripts/ros_driver.py [hunk 1]
```diff
@@ -640,9 +647,13 @@ def run_forever(self):
 
                 if self._target_freq:
                     if rate.remaining().to_sec() <= 0.0:
-                        rospy.logwarn(
-                            f"SMARTS unable to maintain requested target_freq of {self._target_freq} Hz."
-                        )
+                        msg = f"SMARTS unable to maintain requested target_freq of {self._target_freq} Hz."
```
Comment (Gamenot) [inline examples/ros/src/smarts_ros/scripts/ros_driver.py:650]: I think it might be useful to log the timestamp if possible.

**Review by Gamenot — APPROVED** [review]

**Review by liu-allan — APPROVED** [review]

File: examples/ros/src/smarts_ros/scripts/ros_driver.py [hunk 2]
```diff
@@ -121,6 +125,8 @@ def setup_ros(
             assert target_freq > 0.0
             self._target_freq = target_freq
 
+        log_everything_to_ROS(level=logging.WARNING)
```
Comment (liu-allan) [inline examples/ros/src/smarts_ros/scripts/ros_driver.py:128]: if I want to log to ros in the bm code, is this all i need to call in the init of the agent? or would this need to be called in act

File: examples/ros/src/smarts_ros/scripts/ros_driver.py (hunk 1 above)
Comment (sah-huawei) [reply examples/ros/src/smarts_ros/scripts/ros_driver.py:650]: Actually, rospy log messages always include the (ros) timestamp as part of the message.  It's those numbers on the left prior to the message.

File: examples/ros/src/smarts_ros/scripts/ros_driver.py (hunk 2 above)
Comment (sah-huawei) [reply examples/ros/src/smarts_ros/scripts/ros_driver.py:128]: No, actually you shouldn't have to do anything.  This line here will apply to all existing and all future loggers too, including your agent's.  You can look at my current agent for an example of how I configure it's logging.  Once this PR gets merged, those log messages will show up both in the Smarts logfile and in the rospy stream to stdout.

**Pull request merged**